<launch>
  <node name="vis_dlib_cnn" pkg="ros_people_model" type="vis_dlib_cnn.py" output="screen">
    <param name="topic_name" value="/camera/image_raw" />
    <param name="scale" type="double" value="0.2" />
    <param name="fps" type="double" value="1.0" />
    <param name="padding" type="double" value="0.1" />
  </node>

  <node name="vis_dlib_faces" pkg="ros_people_model" type="vis_dlib_faces.py" output="screen">
    <param name="topic_name" value="/vis_dlib_cnn" />
    <param name="scale" type="double" value="0.3" />
    <param name="fps" type="double" value="10.0" />
    <param name="run_face_landmarks" value="false" type="bool" />
    <param name="run_face_id" value="false" type="bool" />
    <param name="run_face_emotions" value="true" type="bool" />
    <param name="run_eye_state" value="false" type="bool" />
  </node>

  <node name="vis_srv_dlib_shapes" pkg="ros_people_model" type="vis_srv_dlib_shapes.py" output="screen" />
  <node name="vis_srv_dlib_id" pkg="ros_people_model" type="vis_srv_dlib_id.py" output="screen" />
  <node name="vis_srv_icog_emopy" pkg="ros_people_model" type="vis_srv_icog_emopy.py" output="screen" />
  <node name="vis_srv_icog_eyestate" pkg="ros_people_model" type="vis_srv_icog_eyestate.py" output="screen" />

  <node name="model_people" pkg="ros_people_model" type="model_people.py" output="screen" />
</launch>
